Changed fictive example to a more generic version.

jellehouwen1 opened the pull request.

jellehouwen1 committed:
Changed fictive example to a more generic version.

tomdrissen commented on src/Examples/HappyFlower.php:
```diff
@@ -78,34 +78,49 @@ public static function fullCard(): Script
         $imageKey = null;
 
-        switch ($zegelCount) {
+        switch ($stampCount) {
```
Refactor to a more generic approach instead of using switch?

tomdrissen reviewed (changes requested):
Add an extra explanation in the README about this example. Maybe we need to hint the user about the images and how to add them. Add a link to the DOCS too.

jellehouwen1 committed:
- Renamed images.
- Moved storage from library to this example.
- Refactored imageKey generation.
- Added more elaborate readme.
- Updated package
